From 4bb9a341d87f10c554d8d8af89add2b0e8c25ff3 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 22 Mar 2020 17:26:35 +0200
Subject: [PATCH] [SW-0] gaudi: pcie: raise delay prior synopsys w/a

The delay before Synopsys W/A in BTL was ~576 usec and in any case it
must be longer than 300 usec.
Set the delay to what it was in BTL.

Change-Id: I66eda721929774796216b6c2adc77c4c0161811d
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/common.c b/gaudi/common.c
index b1d9952..62ddc6f 100644
--- a/gaudi/common.c
+++ b/gaudi/common.c
@@ -79,7 +79,7 @@
 	regmap_writel(0x00402801, mmPCIE_DBI_GEN3_RELATED_OFF);
 
 	/* delay before executing Synopsys W/A > 300us */
-	udelay(300);
+	udelay(576);
 
 	/* Execute the Synopsys W/A */
 	regmap_writel(0xF, mmPCIE_PHY_CR_PARA_CTRL);
